lstchain/image/muon/muon_analysis.py

```diff
@@ -114,9 +113,17 @@ def analyze_muon_event(event_id, image, geom, equivalent_focal_length,
 
     Returns
     ---------
-    muonintensityoutput ``
-    muonringparam       ``
-    good_ring           `bool` it determines whether the ring can be used for analysis or not
+
+    muonintensityoutput MuonEfficiencyContainer
+    ring_size           float, in p.e. total intensity in ring
```

Comment:
to be compliant with the container names in ctapipe, I would suggest to move:
`ring_size` -> `ring_intensity`
`size_outside_ring` -> `intensity_outside_ring`
...
and so on

Reply:
We will soon move to output format common with ctapipe, we can discuss it then.